script/code-gen.sh

```diff
@@ -9,7 +9,7 @@ IMAGE_NAME="kubernetes-codegen:latest"
 CURRENT_DIR="$(pwd)"
```

Comment:
While we are on this file, it could be nice to replace the shebang by `#!/usr/bin/env bash`.